static/scss/footer.scss

```diff
@@ -0,0 +1,158 @@
+// sass-lint:disable mixins-before-declarations
+
+#footer {
```

Comment:
By convention we are using class selectors, do we specifically need to use an id selector here?

Reply:
yeah make sense to be consistent. Moving to class selector here.